### PR TITLE
Feature/HPO create account test case

### PR DESCRIPTION
**Pull request by agrant23**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,3 +136,11 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Mine
+
+sand_box_tools.py
```
Comment (MNadolsky) [inline .gitignore]: something like sand_box*.py

File: coinbase_HPO_createAccount_tests.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+
```
Comment (MNadolsky) [inline coinbase_HPO_createAccount_tests.py]: move this to coinbase_tests.py

File: config.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-
+import os
```
Comment (MNadolsky) [inline config.py]: remove

File: pages/coinbase/homepage.py [hunk 4]
```diff
@@ -51,6 +51,7 @@ def __init__(self, driver, spawn=True, close_cookie_banner=True):
     email_address_field_loc = (by.XPATH, "//input[@type='email']")
     get_started_button_loc =  (
         by.XPATH, "//input[@type='email']/following-sibling::button")
+    account_dial_box_loc =       (by.XPATH, "//span[.='Create account']")
```
Comment (MNadolsky) [inline pages/coinbase/homepage.py]: inconsistent indentation.

don't use the word "dial;" use the full word. There is no memory limit, so just use full variable names _especially_ when you only use them once or twice. Only consider shortening them when they're used frequently. For a reader, it's not worth the cognitive effort of parsing shortened variable names when it's only used once.

File: pages/coinbase/homepage.py [hunk 5]
```diff
@@ -173,6 +174,8 @@ def build_elements(self):
             self.driver, self.email_address_field_loc)
         self.get_started_button =  Button(
             self.driver, self.get_started_button_loc)
+        self.account_dial_box =        Element(
```
Comment (MNadolsky) [inline pages/coinbase/homepage.py]: inconsistent indention. 

don't abbreviate words in variable names.

**Review by MNadolsky — CHANGES_REQUESTED** [review]

File: coinbase_tests.py [hunk 6]
```diff
@@ -151,9 +151,49 @@ def tearDown(self):
         self.page.driver.quit()
 
 
+class HomePageSetup(unittest.TestCase):
```
Comment (MNadolsky) [inline coinbase_tests.py]: use "template" in template class names to distinguish from class names.

File: coinbase_tests.py [hunk 7]
```diff
@@ -151,9 +151,49 @@ def tearDown(self):
...
 
+    def setUp(self):
+        self.driver = webdriver.Chrome(root_path + config.win_chromedriver_path)
```
Comment (MNadolsky) [inline coinbase_tests.py]: incompatible with mac

File: pages/coinbase/homepage.py [hunk 8]
```diff
@@ -200,6 +209,16 @@ def build_elements(self):
         self.litecoin_buy_button = Button(
             self.driver, self.litecoin_buy_button_loc)
 
+        self.total_product_link_list = [
```
Comment (MNadolsky) [inline pages/coinbase/homepage.py]: scrub all unused code from the branch

**Review by MNadolsky — CHANGES_REQUESTED** [review]

File: coinbase_tests.py (hunk 7 above)
Comment (agrant23) [reply coinbase_tests.py]: If you check this PR first check my comment in the PR for feature/HPOsafeSecure

**Review by MNadolsky — APPROVED** [review]

**Pull request merged**